app/models/nic/base.rb

```diff
@@ -124,11 +124,6 @@ class Jail < ::Safemode::Jail
         :children_mac_addresses, :nic_delay, :fqdn, :shortname, :type, :managed?, :bond?, :bmc?, :provision?
     end
 
-    # include STI inheritance column in audits
```

Comment:
Hmm, re-visiting this, I think that's not a correct solution :/ Although, I didn't find this method being overridden for other STIs...